e2e/e2e_test.go

```diff
@@ -284,3 +566,39 @@ func inRange(cidr string, ip string) error {
 
 	return fmt.Errorf("ip [%s] is NOT in range %s", ip, cidr)
 }
+
+func runIpReconciler(kubeconfigPath string) error {
+	var err error
+	var ipReconcileLoop *reconciler.ReconcileLooper
+
+	reconcilerTimeout := defaultReconcilerTimeout
+	logLevel := "error"
+
+	logging.SetLogLevel(logLevel)
+
+	if kubeconfigPath == "" {
+		ipReconcileLoop, err = reconciler.NewReconcileLooper(context.Background(), reconcilerTimeout)
+	} else {
+		ipReconcileLoop, err = reconciler.NewReconcileLooperWithKubeconfig(context.Background(), kubeconfigPath, reconcilerTimeout)
+	}
+	if err != nil {
+		_ = logging.Errorf("failed to create the reconcile looper: %v", err)
+		os.Exit(couldNotStartOrphanedIPMonitor) // not sure if i should be exiting or returning the error?
```

Comment:
yeah, `os.Exit` shouldn't be used. Just return an error - `return fmt.Errorf("failed to create the reconcile looper: %w", err)` .

Remove the previous logging line as well.